Spigot-Server-Patches/0003-MC-Dev-fixes.patch

```diff
@@ -229,8 +260,57 @@ index 86585a13ff..f52b9c6f0a 100644
          for (int i = 0; i < afloat.length; ++i) {
              afloat[i] = (float) Math.sin((double) i * 3.141592653589793D * 2.0D / 65536.0D);
          }
+diff --git a/src/main/java/net/minecraft/server/ProtoChunk.java b/src/main/java/net/minecraft/server/ProtoChunk.java
+index 824e68b50..d0bb115eb 100644
+--- a/src/main/java/net/minecraft/server/ProtoChunk.java
++++ b/src/main/java/net/minecraft/server/ProtoChunk.java
+@@ -65,7 +65,7 @@ public class ProtoChunk implements IChunkAccess {
+         this.b = chunkcoordintpair;
+         this.p = chunkconverter;
+         Predicate predicate = (block) -> {
+-            return block == null || block.getBlockData().isAir();
++            return block == null || ((Block)block).getBlockData().isAir(); // Paper - decompile fix
```

Comment:
Add space after cast. The `predicate` variable is reused.

Reply:
will do